src/app/modules/sgid/sgid.constants.ts

```diff
@@ -33,7 +33,18 @@ export enum SGIDScope {
   DateOfBirth = 'myinfo.date_of_birth',
   PassportNumber = 'myinfo.passport_number',
   PassportExpiryDate = 'myinfo.passport_expiry_date',
-  MobileNumber = 'myinfo.mobile_number',
   Email = 'myinfo.email',
   RegisteredAddress = 'myinfo.registered_address',
+  Sex = 'myinfo.sex',
+  Race = 'myinfo.race',
+  Nationality = 'myinfo.nationality',
+  HousingType = 'myinfo.housingtype',
+  HdbType = 'myinfo.hdbtype',
+  BirthCountry = 'myinfo.birth_country',
+  VehicleNo = 'myinfo.vehicles',
+  Employment = 'myinfo.name_of_employer',
+  WorkpassStatus = 'myinfo.workpass_status',
+  WorkpassExpiryDate = 'myinfo.workpass_expiry_date',
+  MaritalStatus = 'myinfo.marital_status',
+  MobileNoWithPrefix = 'myinfo.mobile_number_with_prefix', // SGID also has another myinfo.mobile_number field that does not contain the country code prefix
```

Comment:
Oh so we are only getting the one with prefix right?

Reply:
yup correct! I edited the comment to make it more explicit [d468515](https://github.com/opengovsg/FormSG/pull/6766/commits/d4685150ecdbd802e2a9d7bb516e8d52adca83a8)